base/exports.jl

```diff
@@ -1165,10 +1165,9 @@ export
     RawFD,
     read,
     read!,
-    readall,
+    readstring,
     readavailable,
     readbytes!,
```

Comment:
See explanation in commit message: https://github.com/samoconnor/julia/commit/066c27e3aa3628b218ef68dc3f5db956f68f0e87